vpn/install2.sh

```diff
@@ -1,24 +1,20 @@
 #!/bin/bash
```

Comment:
There is an old bug where install2.sh was called twice, the original code here was to ensure things are in good order.  The file size check was to do that.  When the bug occurs, the server.conf was set to 0

Reply:
File size check is moved to confgen.sh, which should do the same thing. install2.sh will only generate key chains for server.